torch/nn/quantized/functional.py

```diff
@@ -397,6 +397,17 @@ def elu(input, alpha=1., inplace=False, scale=None, zero_point=None):
     else:
         return torch._C._nn.elu(input, alpha)
 
+def hardsigmoid(input):
+    # type: (Tensor) -> Tensor
+    r"""
+    Applies the quantized element-wise function :math:`\text{Hardsigmoid}(x) = \frac{ReLU6(x + 3)}{6}`
```

Comment:
Please, change the math documentation to the `cases` -- using `ReLU6` in the math description is not very easy to understand.